react-native/setup/deploy-project-to-production.md

```diff
@@ -0,0 +1,65 @@
+# [MO] Setup & Deploy Project to Production - Only iOS Testflight
+
+## Owner: [Xavier Lefèvre](github.com/xavierlefevre)
+
+## Prerequisites
+- [ ] [React Native Toolbox](https://github.com/bamlab/generator-rn-toolbox)
+- [ ] Your client admin account on Apple Developer
+- [ ] Your client admin account on iTunes Connect
+
+## Steps
+
+### 1. Setup Production Env
+> Creates the necessary fastlane and JS environments files
+```bash
+yo rn-toolbox:fastlane-env
+```
+
+***Answers***
+- Please confirm the project name: `<Press Enter>` => The same as for the staging/hockeyapp setup
+- The name for this new environment (lowercase, no space): `production`
+- The name of your repository Git branch for the environment just set: `production` => Or master...
+- The name of the company which will be publishing this application: `Bam` => Or your client name
+- The app name for this environment: `<AppName>` => The final App name
+- The App Id for this environment: `com.bnp.<ProjectName>.production`
+- Which platform will you use for deployment?: `AppStore`
+- Your git repo for match: `git@github.com:bamlab/certificates.git`
```

Comment:
... yes, pas pensé, mais c'est aussi sur le MO deploy to staging :)
je l'enlève